src/components/global/Widget/index.tsx

```
@@ -0,0 +1,59 @@
+    }
+  }, [user]);
+
+  return (
+    <div className="p-5">
+      <ClerkLoading>
+        <div className="h-full flex justify-center items-center">
+          <Loader />
+        </div>
+      </ClerkLoading>
+      <SignedIn>
+        {profile && profile.user ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)